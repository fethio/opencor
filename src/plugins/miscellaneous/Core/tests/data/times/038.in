<math xmlns="http://www.w3.org/1998/Math/MathML">
    <apply>
        <eq/>
        <ci>a</ci>
        <apply>
            <divide/>
            <apply>
                <times/>
                <apply>
                    <minus/>
                    <apply>
                        <minus/>
                        <ci>b</ci>
                        <ci>c</ci>
                    </apply>
                </apply>
                <ci>d</ci>
            </apply>
            <exponentiale/>
        </apply>
    </apply>
</math>
