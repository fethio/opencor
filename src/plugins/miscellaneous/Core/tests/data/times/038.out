<math xmlns="http://www.w3.org/1998/Math/MathML">
    <mrow>
        <mi mathvariant="italic">a</mi>
        <mo>=</mo>
        <mfrac>
            <mrow>
                <mo>−</mo>
                <mo>(</mo>
                <mi mathvariant="italic">b</mi>
                <mo>−</mo>
                <mi mathvariant="italic">c</mi>
                <mo>)</mo>
                <mo>·</mo>
                <mi mathvariant="italic">d</mi>
            </mrow>
            <mi>e</mi>
        </mfrac>
    </mrow>
</math>
